doc/api/buffer.markdown

```diff
@@ -130,10 +130,7 @@ the list together.
 If the list has no items, or if the totalLength is 0, then it returns a
 zero-length buffer.
 
-If the list has exactly one item, then the first item of the list is
-returned.
-
-If the list has more than one item, then a new Buffer is created.
+If the list has one or more items, then a new Buffer is created.
```

Comment:
new Buffer is created in any case, even if the list has zero items, so this statement could be misleading.

Maybe change it to an assurance like "it'll always return a new Buffer instance, which you can mutate all you want".